doc/nvim-tree-lua.txt

```diff
@@ -1406,6 +1406,7 @@ NvimTreeSymlink
 NvimTreeFolderName          (Directory)
 NvimTreeRootFolder
 NvimTreeFolderIcon
+NvimTreeOpenedFolderIcon
```

Comment:
We should be explicit about the default:

`NvimTreeOpenedFolderIcon     (NvimTreeFolderIcon)`